From 02a3e3fc5e7abda3dd44b3c89c4318c16869e873 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Wed, 7 Feb 2018 18:57:17 +0800
Subject: [PATCH] media/video/generic_sensor: fix compile warning

drivers/media/video/generic_sensor.c:865:5: warning: this if
clause does not guard... [-Wmisleading-indentation]

Change-Id: Ic34e59cb45b28864ccc15f5c641cd6aedbff4dab
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 drivers/media/video/generic_sensor.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/video/generic_sensor.c b/drivers/media/video/generic_sensor.c
index 19ebedddaf80..767076256fe5 100644
--- a/drivers/media/video/generic_sensor.c
+++ b/drivers/media/video/generic_sensor.c
@@ -865,7 +865,7 @@ int generic_sensor_init(struct v4l2_subdev *sd, u32 val)
     if (sensor->flash_off_timer.timer.function==NULL)
         sensor->flash_off_timer.timer.function = generic_flash_off_func;
     
-	sensor->info_priv.funmodule_state |= SENSOR_INIT_IS_OK;
+    sensor->info_priv.funmodule_state |= SENSOR_INIT_IS_OK;
     
 	return 0;
 sensor_INIT_ERR:
-- 
2.35.3

